app/report/rest.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+
+from flask import Blueprint, current_app, jsonify, request
+
+from app.dao.reports_dao import create_report
+from app.dao.services_dao import dao_fetch_service_by_id
+from app.errors import register_errors
+from app.models import Report, ReportStatus, ReportType
+from app.schema_validation import validate
+
+report_blueprint = Blueprint("report", __name__, url_prefix="/service/<uuid:service_id>/report")
+register_errors(report_blueprint)
+
+
+@report_blueprint.route("", methods=["POST"])
+def create_service_report(service_id):
+    "Creates a new report for a service"
+
+    data = request.get_json()
+
+    validate(data, {"report_type": {"type": "string", "required": True}})
+
+    # Validate report type is one of the allowed types
+    report_type = data.get("report_type")
+    if report_type not in [rt.value for rt in ReportType]:
```

Comment:
You may be planning this for future iterations, but what are your thoughts on leveraging schema validation here, similar to [what we do with templates](https://github.com/cds-snc/notification-api/blob/d7cede9b256aea06ff19c7ded09a4fe593c1ac49/app/schemas.py#L420)?

Reply:
Good idea! I'll add that in my next PR